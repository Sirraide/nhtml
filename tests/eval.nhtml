div[bla=12345] {
    /foobar[foo=bar]#baz (Baz)
    /foobar[foo=bar]#baz (Baz)
    /foobar[foo=bar]#baz (Baz)
    /foobar[foo=bar]#baz (Baz)
    /foobar[foo=bar]#baz (Baz)
}

eval {
    $('div').attributes = {foo : 'bar', baz : 'qux', quux : 'quuz'}
    $('a').id = 1
    for (const ch of $('div').children) delete ch.attributes.href
}
