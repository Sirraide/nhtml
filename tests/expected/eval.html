<div baz='qux' foo='bar' quux='quuz'>
    <a id='1' foo='bar'>Baz</a>
    <a id='baz' foo='bar'>Baz</a>
    <a id='baz' foo='bar'>Baz</a>
    <a id='baz' foo='bar'>Baz</a>
    <a id='baz' foo='bar'>Baz</a>
</div>
